src/TextArea/TextArea.less

```diff
@@ -79,6 +82,10 @@
   color: @neutral_dark_gray;
   background: @neutral_silver;
 
+  .TextArea--label {
```

Comment:
Hmm, actually, this specific case of nesting seems pretty necessary since it's a style that should only be applied when the component is disabled